content/ngf/installation/installing-ngf/deploy-data-plane.md

```diff
@@ -0,0 +1,243 @@
+product: NGF
+docs: DOCS-000
+---
+
+## Overview
+
+This document describes how to use a Gateway to deploy the NGINX data plane, and how to modify it using an NGINX custom resource.
+
+[A Gateway](https://gateway-api.sigs.k8s.io/concepts/api-overview/#gateway) is used to manage all inbound requests, and is a key Gateway API resource.
+
+When a Gateway is attached to a GatewayClass associated with NGINX Gateway Fabric, it creates a Service and an NGINX deployment. This forms the NGINX data plane, handling requests.
+
+A single GatewayClass can have multiple Gateways: each Gateway will create a separate Service and NGINX deployment.
+
+## Before you begin
+
+- [Install]({{< ref "/ngf/installation/" >}}) NGINX Gateway Fabric.
+
+## Create a Gateway
+
+To deploy a Gateway, run the following command:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: cafe
+spec:
+  gatewayClassName: nginx
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+EOF
+```
+
+To check that the Gateway has deployed correctly, use `kubectl describe` to check its status:
+
+```shell
+kubectl describe gateway
+```
+
+You should see these conditions:
+
+```text
+Conditions:
+      Last Transition Time:  2025-05-05T23:49:33Z
+      Message:               Listener is accepted
+      Observed Generation:   1
+      Reason:                Accepted
+      Status:                True
+      Type:                  Accepted
+      Last Transition Time:  2025-05-05T23:49:33Z
+      Message:               Listener is programmed
+      Observed Generation:   1
+      Reason:                Programmed
+      Status:                True
+      Type:                  Programmed
+      Last Transition Time:  2025-05-05T23:49:33Z
+      Message:               All references are resolved
+      Observed Generation:   1
+      Reason:                ResolvedRefs
+      Status:                True
+      Type:                  ResolvedRefs
+      Last Transition Time:  2025-05-05T23:49:33Z
+      Message:               No conflicts
+      Observed Generation:   1
+      Reason:                NoConflicts
+      Status:                False
+      Type:                  Conflicted
+```
+
+Using `kubectl get` you can see the NGINX Deployment:
+
+```shell
+kubectl get deployments
+```
+```text                                                                                                                                    ⎈ kind-kind
+NAME         READY   UP-TO-DATE   AVAILABLE   AGE
+cafe-nginx   1/1     1            1           3m18s
+```
+
+You can also see the Service fronting it:
+
+```shell
+kubectl get services
+```
+```text                                                                                                                                        ⎈ kind-kind
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
+cafe-nginx   NodePort    10.96.125.117   <none>        80:30180/TCP   5m2s
```

Comment:
Since LoadBalancer is the default, maybe we show that here instead. And then below we can change to something else.